charts/otel-cloud-stack/values.yaml

```diff
@@ -84,9 +85,15 @@ daemonCollector:
         root_path: /hostfs
         scrapers:
           cpu:
-          disk:
-          load:
+            metrics:
+              system.cpu.utilization:
+                enabled: true
+          disk: {}
+          load: {}
           filesystem:
```

Comment:
```suggestion
          filesystem: {}
```

Reply:
oh nevermind

Reply:
filesystem doesn't need that because it has values set for it already